modules/mobile/config/routes.rb

```diff
@@ -68,6 +68,10 @@
           resources :threads, only: %i[index]
         end
 
+        resources :threads, defaults: { format: :json } do
+          patch :move, on: :member
+        end
```

Comment:
I would tweak the way you write this route as it's generating more routes than are used. You can change
```
resources :threads, defaults: { format: :json } do
```
to
```
resources :threads, only: %I[], defaults: { format: :json } do
```
It's weird syntax but it prevents rails from generating full crud urls (index, show, new, create, edit, update, destroy) for the `thread` resource

Reply:
Excellent catch. This was copied from the upstream team and it looks like they're being sloppy with their routes. But in this case, I think we should nest the `move` method within the resources on line 68 that Jason recently created. The folder id is required for this to work, so there's no good reason to make it a query param and not a route param.